phoenix-core/src/main/java/org/apache/phoenix/hbase/index/IndexRegionObserver.java

```diff
@@ -824,39 +781,7 @@ private void doPre(ObserverContext<RegionCoprocessorEnvironment> c, BatchMutateC
 
   @Override
   public void postOpen(final ObserverContext<RegionCoprocessorEnvironment> c) {
-    Multimap<HTableInterfaceReference, Mutation> updates = failedIndexEdits.getEdits(c.getEnvironment().getRegion());
-
-    if (this.disabled) {
-        return;
-    }
-
-    long start = EnvironmentEdgeManager.currentTimeMillis();
-    try {
-        //if we have no pending edits to complete, then we are done
-        if (updates == null || updates.size() == 0) {
-          return;
-        }
-
-        LOG.info("Found some outstanding index updates that didn't succeed during"
-                + " WAL replay - attempting to replay now.");
-
-        // do the usual preWriter stuff
-        try {
-            preWriter.writeAndHandleFailure(updates, true, ScanUtil.UNKNOWN_CLIENT_VERSION);
-        } catch (IOException e) {
-                LOG.error("During WAL replay of outstanding index updates, "
-                        + "Exception is thrown instead of killing server during index writing", e);
-        }
-    } finally {
-         long duration = EnvironmentEdgeManager.currentTimeMillis() - start;
-         if (duration >= slowPostOpenThreshold) {
-             if (LOG.isDebugEnabled()) {
-                 LOG.debug(getCallTooSlowMessage("postOpen", duration, slowPostOpenThreshold));
-             }
-             metricSource.incrementNumSlowPostOpenCalls();
-         }
-         metricSource.updatePostOpenTime(duration);
-    }
+      return;
```

Comment:
If you're removing all the logic from this coprocessor hook you could just go ahead and remove the method. I believe the superclass has a default no-op implementation.